PR: https://github.com/odoo/odoo/pull/142172

From: 1de2bbd325cb1c0fc8628569b61a86bede466599
From: svs-odoo
Date: 2023-11-20 17:17:16

Structural Changes: 3
Total Changes: 8

[IMP] repair, stock(_picking_batch): clean

Removes the fields `show_set_qty_button` and `show_clear_qty_button` and
their compute method from the models `repair.order`, `stock.picking` and
`stock.picking.batch`. Also, simplify the invisible condition on the
`action_assign` button for the Repair Order.

closes odoo/odoo#142172

Related: odoo/upgrade#5368
Signed-off-by: William Henrotin (whe) <whe@odoo.com>
Signed-off-by: Steve Van Essche <svs@odoo.com>

================================= pseudo patch: =================================

--- a/addons/stock_picking_batch/models/stock_picking_batch.py
+++ b/addons/stock_picking_batch/models/stock_picking_batch.py
@@ -57,14 +57,6 @@ class StockPickingBatch(models.Model):
               - If not manually changed and transfers are added/removed/updated then this will be their earliest scheduled date
                 but this scheduled date will not be set for all transfers in batch.""")
     is_wave = fields.Boolean('This batch is a wave')
-    # To remove in master
-    show_set_qty_button = fields.Boolean(compute='_compute_show_qty_button')
-    show_clear_qty_button = fields.Boolean(compute='_compute_show_qty_button')
-
-    @api.depends()
-    def _compute_show_qty_button(self):
-        self.show_set_qty_button = False
-        self.show_clear_qty_button = False
 
     @api.depends('company_id', 'picking_type_id', 'state')
     def _compute_allowed_picking_ids(self):
